core/src/main/java/com/hcl/ecomm/core/config/MagentoServiceConfig.java

```diff
@@ -0,0 +1,31 @@
+package com.hcl.ecomm.core.config;
+
+import org.osgi.service.metatype.annotations.AttributeDefinition;
+import org.osgi.service.metatype.annotations.AttributeType;
+import org.osgi.service.metatype.annotations.ObjectClassDefinition;
+
+@ObjectClassDefinition(name = "Magento API's Service Configuration")
+public @interface MagentoServiceConfig {
+    String domainNameDefaultValue = "localhost/magento2/rest";
```

Comment:
@DeepaliAgrwl : This should come from login config

Reply:
change incorporated